java/vertx4/pom.xml

```
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.vertx4</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+    <name>vertx4</name>
+    <description>Demo project for Vertx</description>
+
+    <properties>
+        <java.version>11</java.version>
+        <vertx.version>4.0.0</vertx.version>
```

better use a range, to ensure we test the last version (as specify on `config.yaml`)
```suggestion
        <vertx.version>[4.0,4.1)</vertx.version>
```

---

Vert.x 4.1 does not exist

---

`[4.0,4.1)` means >= 4.0 and <= 4.1